tslint.json

```diff
@@ -64,7 +63,7 @@
       "ignore-params"
     ],
     "no-misused-new": true,
-    "no-non-null-assertion": true,
+    "no-non-null-assertion": false,
```

Comment:
`!` is sometimes needed in strict mode. Since we will switch to it, I anticipate